src/bots/liquidator.ts

```diff
@@ -1684,6 +1716,14 @@ tx: ${tx} `
 			const auth = userAcc.authority.toBase58();
 			const userKey = user.userAccountPublicKey.toBase58();
 
+			if (this.excludedAccounts.has(userKey)) {
+				// Debug log precisely because the intent is to avoid noise
+				logger.debug(
+					`Skipping liquidation for ${userKey} due to configuration`
+				);
+				continue;
+			}
+
```

Comment:
Thanks. I thought about doing it there, but was aiming to minimize the amount of code changed, and I didn't want to either call `userAcc.authority.toBase58()` twice or expand the array of returned values to include it.

I guess it makes little difference, since it'll need to end up being called during the next loop anyway.